src/components/MediaUploader.vue

```diff
@@ -175,33 +187,19 @@ export default {
         },
 
         generateUuid() {
```

Comment:
Quite expensive algorithm, could potentially use an incremented integer for this.